apis/nucleus/src/components/Cell.jsx

```diff
@@ -290,17 +290,40 @@ const Cell = forwardRef(
     const [selections] = useObjectSelections(app, model);
     const [hovering, setHover] = useState(false);
     const hoveringDebouncer = useRef({ enter: null, leave: null });
+    const [bgColor, setBgColor] = useState(undefined);
     const focusHandler = useRef({
       focusToolbarButton(last) {
         // eslint-disable-next-line react/no-this-in-sfc
         this.emit(last ? 'focus_toolbar_last' : 'focus_toolbar_first');
       },
     });
 
+    const resolveBgColor = () => {
+      const bgComp = layout && layout.components ? layout.components.find((comp) => comp.key === 'general') : null;
+
+      if (bgComp && bgComp.bgColor && bgComp.bgColor.useColorExpression) {
+        if (bgComp.bgColor.colorExpression) {
+          bgComp.bgColor.colorExpression = { color: bgComp.bgColor.colorExpression };
```

Comment:
We should parse the color here as well, bascially use d3-color. I'm gonna check if we should expose that from "theme" maybe.

Reply:
I guess d3-color is being used by "theme", would I need to add it in nucleus as well, or should we wait for the exposure from theme?